src/test/java/com/wikia/webdriver/elements/mercury/pages/discussions/ReportedPostsAndRepliesPage.java

```diff
@@ -0,0 +1,19 @@
+package com.wikia.webdriver.elements.mercury.pages.discussions;
+
+import com.wikia.webdriver.elements.mercury.components.discussions.common.ErrorMessages;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import lombok.Getter;
+
+public class ReportedPostsAndRepliesPage extends WikiBasePageObject {
+
+  @Getter(lazy = true)
+  private final ErrorMessages errorMessages = new ErrorMessages();
+
+  private static final String PATH = "d/reported";
+
+  public ReportedPostsAndRepliesPage open() {
+    driver.get(urlBuilder.getUrlForWiki().replace("/wiki", "") + String.format(PATH));
```

Comment:
That looks like a hack :)
Can't we have a method that returns wikiDomain? 
It could work as:
```urlBuilder.getUrlForWiki().replace("/wiki", "") + PATH```